docs/deploy/arm_compute_lib.rst

```diff
@@ -26,6 +26,24 @@ and GPU's. Currently the integration offloads operators to ACL to use hand-craft
 routines in the library. By offloading select operators from a relay graph to ACL we can achieve
 a performance boost on such devices.
 
+Installing Arm Compute Library
+------------------------------
+
+We recommend two different ways to build and install ACL:
+
+* Use the script located at `docker/install/ubuntu_install_arm_compute_library.sh`. You can use this
+  script for building ACL from source natively or for cross-compiling the library on an x86 machine.
+  You may need to change the architecture of the device you wish to compile for by altering the
+  `target_arch` variable. Binaries will be built from source and installed to the location denoted by
+  `install_path`.
+* Alternatively, you can download and use pre-built binaries from:
+  https://github.com/ARM-software/ComputeLibrary/releases. When using this package, you will need to
+  select the binaries for the architecture you require and make sure they are visible to cmake.
```

Comment:
Need to be more specific, like replace `lib` with `lib/<target-archi>`.

Reply:
Yeah, I think it would be nicer it we can just provide some simple commands.

Reply:
Yep I can add this, my intention was to leave it a bit more open as this may change in the future, although this can always be updated